db/dcrpg/pgblockchain.go

```diff
@@ -606,34 +606,35 @@ func (pgb *ChainDB) VersionCheck(client *rpcclient.Client) error {
 		log.Debugf("Table %s: v%s", tab, ver)
 	}
 
-	if tableUpgrades := TableUpgradesRequired(vers); len(tableUpgrades) > 0 {
-		if tableUpgrades[0].UpgradeType == "upgrade" || tableUpgrades[0].UpgradeType == "reindex" {
-			// CheckForAuxDBUpgrade makes db upgrades that are currently supported.
-			isSuccess, err := pgb.CheckForAuxDBUpgrade(client)
-			if err != nil {
-				return err
-			}
-			// Upgrade was successful, no need to proceed.
-			if isSuccess {
-				return nil
-			}
-		}
+	needsUpgrade := make([]TableUpgrade, 0)
```

Comment:
Why `make([]TableUpgrade, 0)`?  in terms of appending, it's equally valid to append to a nil slice and a non-nil zero length slice.  Did you mean to do `make([]TableUpgrade, 0, len(vers))`

Reply:
I always forget this.